index.html

```diff
@@ -31,8 +31,8 @@
       </ul>
       <a href="#" class="navbar-menu-button"><img src="./assets/svg/menu.svg" alt="Menu" /></a>
     </nav>
-    <header class="header container">
-      <section class="header-content">
+    <header class="header">
+      <section class="header-content container">
         <h1 class="header-title font-crete-round text-orange">Hey There. I'm Aimal</h1>
```

Comment:
- [ ] Good job so far 🔥 your design looks great and it follows the template. Kindly align the social links to the right in the desktop version as shown in the template 👍 
- Your design:
![Annotation 2022-03-23 195948](https://user-images.githubusercontent.com/85113567/159777399-2b95023d-f61e-457e-a473-53e962b3c0a1.jpg)
- Figma design:
![Annotation 2022-03-23 200000](https://user-images.githubusercontent.com/85113567/159777650-f8ccbaed-3372-4ae0-ba18-fac74c94ca94.jpg)